core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -86,6 +126,21 @@ class InvokerPool(
 
         case GetStatus => sender() ! invokerStatus.toMap
 
+        case msg: InvocationFinishedMessage => {
+            // Forward message to invoker, if InvokerActor exists
+            invokers.get(msg.name).map(_.forward(msg))
+        }
+
+        case msg: ActivationRequest => {
```

Comment:
I think under load this could become a bottleneck. This change makes me nervous  without data to show otherwise, since both the request and active acks are funneled through the same actor.